src/components/ContainerHeader.vue

```diff
@@ -29,23 +29,16 @@
                         <i class="fa fa-info" aria-hidden="true"/>
                     </a>
                 </div>
-                <div
-                    v-if="sidebarState.sidebarSection === 'user'"
-                    class="kiwi-header-option kiwi-header-option-user kiwi-header-option--active"
-                >
-                    <a @click="sidebarState.close()">
-                        <i class="fa fa-user" aria-hidden="true"/>
-                    </a>
-                </div>
                 <div
                     :class="{
                         'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist'
+                            || sidebarState.sidebarSection === 'user'
                     }"
                     class="kiwi-header-option kiwi-header-option-nicklist"
                 >
                     <a
                         :title="$t('person', {count: Object.keys(buffer.users).length})"
-                        @click="sidebarState.toggleNicklist()"
+                        @click="toggleNicklist()"
```

Comment:
Having this as a method seems inconsistent, feel this should be reverted and the method removed.